app/styles/quickgo.scss

```diff
@@ -153,6 +153,33 @@ table th{
   background-color: $primary5;
 }
 
+.your-selection-container a.icon-functional{
+  border-bottom: 0px;
+}
+
+.removedTerms{
+  border: 1px solid #666;
+  background-color: $verylightgrey;
+  padding: 10px;
+  border-radius: 10px;
```

Comment:
Do we need a whole new different style for this component? To discuss with @SangyaP